packages/dai-plugin-migrations/test/helpers/ganache.js

```diff
@@ -0,0 +1,31 @@
+import fetch from 'node-fetch';
```

Comment:
could we move helpers like these higher up in the monorepo? ignore me if this feels like premature optimization

Reply:
or into a new directory in `packages/`, that just gets used locally, not published to npm? is that something lerna supports or not really?

Reply:
Yea, totally agree that we can create a little micro package. It is something lerna supports - it's just a package w/ private marked to `true`.

I was thinking of tackling it as part of another PR though, to keep things simpler :)

Reply:
great! and thanks for keeping the PR focused 🙏